cmd/dump.go

```diff
@@ -70,10 +72,10 @@ func runDump(ctx *cli.Context) error {
 	}
 
 	tmpDir := ctx.String("tempdir")
-	if _, err := os.Stat(tmpDir); os.IsNotExist(err) {
+	if _, err := fs.AppFs.Stat(tmpDir); os.IsNotExist(err) {
```

Comment:
I don't think we need to change tmpdir to abstract filesystem. We should still use local file system as tempdir.

Reply:
If you use local filesystem for tmpdir, then `fs.AppFs.Stat` will fail if you use other abstract filesystem layer. That’s why afero offers the synonym `TempDir` like os.

Reply:
I mean just keep `os.Stat(tmpdir)` because we only need to move `avatars/attachments/lfs objects` to an abstract file system, but not all files.

Reply:
@lunny ah ok, so I think we don't need almost of the afero change, just to keep the one in `modules/storage.go`